concourse/pipelines/concourse-lite-self-terminate.yml

```diff
@@ -0,0 +1,31 @@
+resources:
+  - name: delete-timer
+    type: time
+    source:
+      start: 20:00 -0000
+      stop: 21:00 -0000
```

Comment:
Out of interest, why is the `stop` time for this earlier than the `stop` time in `autodelete-cloudfoundry.yml`?